README.md

```diff
@@ -156,23 +220,35 @@ The column will display the contents of the results of the Autogov status (pass/
    ```zsh
    # from root
    yarn --cwd packages/app add @liatrio/backstage-plugin-autogov-status-catalog-column@^1.6.2
-   yarn --cwd @liatrio/backstage-plugin-backend-module-autogov-processor@^1.6.2
+   yarn --cwd packages/backend add @liatrio/backstage-plugin-backend-module-autogov-processor@^1.6.2
    ```
 
 #### Autogov Plug-in Latest Release Autogov Status Catalog Processor
 
 1. Update `packages/backend/src/index.ts` with:
 
+`diff`
+
 ```diff
 +  backend.add(
 +    import('@liatrio/backstage-plugin-backend-module-autogov-processor'),
 +  );
 backend.start();
 ```
 
+`copy paste`
+
+```.ts
+backend.add(
+  import('@liatrio/backstage-plugin-backend-module-autogov-processor'),
+);
+```
+
 #### Autogov Plug-in Latest Release Autogov Status Catalog Column
 
-1. Update `packages/app/App.tsx`:
+1. Update `packages/app/src/App.tsx`:
+
+`diff`
```

Comment:
If this is indented with tabs to be sub-level to the above `1.`, the numbering sequence will be automatically continued.